functions/wcif.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+
+const ClearWCIF = {
+  name: 'ClearWCIF',
+  // FIXME: quick helper to make sure we start "from scratch", maybe also
+  // clean other properties/extensions which I'm not aware of?
+  docs: 'Remove all childActivities keeping only the main schedule, remove all assignments.',
+  args: [],
+  outputType: 'Void',
+  usesContext: true,
+  mutations: ['schedule', 'persons'],
+  implementation: (ctx, filename) => {
+    ctx.competition.persons.forEach((person) => {
+      // FIXME: should we clean up "roles" too actually?
+      // Changing "delegate"/"organizers"/"trainee-delegate" would have no effect
+      // as they are - expectedly! - not patchable on the WCA, but any "staff-*"
+      // should likely be removed too.
+      person.assignments = []
+    })
+    ctx.competition.schedule.venues.forEach((venue) => {
+      venue.rooms.forEach((room) => {
+        room.extensions = []
+        room.activities.forEach((activity) => {
+          activity.extensions = []
+          activity.childActivities = []
+        })
+      })
+    })
+  }
+}
+
+const ImportWCIF = {
+  name: 'ImportWCIF',
+  docs: 'Import the WCIF from a json file',
+  args: [
+    {
+      name: 'filename',
+      type: 'String',
+      docs: 'WCIF filename (relative to SCRIPT_BASE)',
+      defaultValue: 'input.json',
```

Comment:
Maybe don't use a defaultValue ? I think it would be better to be explicit.

Reply:
Sounds good to me, I'll update this!